src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/BaseClaimPdfTest.java

```diff
@@ -33,7 +33,13 @@ public void shouldBeAbleToFindTestClaimDataInSealedClaimPdf() throws IOException
     protected void shouldBeAbleToFindTestClaimDataInPdf(String pdfName) throws IOException {
         Claim createdCase = createCase();
         String pdfAsText = textContentOf(retrievePdf(pdfName, createdCase.getExternalId()));
-        assertionsOnClaimPdf(createdCase, pdfAsText);
+        assertionsOnPdf(createdCase, pdfAsText);
+    }
+
+    protected void shouldBeAbleToFindTestCCJDataInPdf(String pdfName) throws IOException {
```

Comment:
I would inline this into the test method, its not being reused currently it doesn't really make sense being here